### PR TITLE
TM-5602-Move-Action

### PR DESCRIPTION
**Pull request by cagbakwu**
[Ticket](https://metaphase.atlassian.net/browse/TM-5602)
Screenshot 2024-02-21 at 2 42 12 PM
Screenshot 2024-02-21 at 2 42 53 PM

### CODE REVIEW

**Comment by cagbakwu** [comment]
Only 4 files, the other 4 are snapshots.

**Comment by elizabeth-jimenez** [comment]
We have some CSS that's specific to which row is where that you'll definitely want to update, as well

**Comment by elizabeth-jimenez** [comment]
image

**Review by elizabeth-jimenez — CHANGES_REQUESTED** [review]
1. in _variables.scss go through the 6 configs for Agenda Item Maintenance Legs and update, as needed.
2. in AgendaLeg.jsx update onHover$()

**Comment by cagbakwu** [comment]
Had to update _agendaItemRow as well 👍

**Comment by elizabeth-jimenez** [comment]
- [ ]  agenda item maintenance read
- [ ]  AIM edit
- [ ]  panel agenda item
- [ ]  print view of panel meeting agenda
- [ ]  agenda item history

**Comment by elizabeth-jimenez** [comment]
image
seems like a lot of space between Action and the top X row

**Comment by elizabeth-jimenez** [comment]
image
AIM Edit - not sure if this was already on dev branch tho, i can go verify in a bit

File: src/sass/_variables.scss [hunk 1]
```diff
@@ -299,13 +299,13 @@ $aim-legs-sm-height-config: (
...
-  "VICE": 14,
-  "Pay Plan": 15
+  "VICE": 13,
+  "Pay Plan": 14
 );
 $aim-legs-lrg-height-config: (
   "Language": 6,
-  "Action": 12,
-  "Travel": 13,
+  "Action": 1,
```
Comment (elizabeth-jimenez) [inline src/sass/_variables.scss:307]: Action shouldn't be 1 here, look at the rendered css (dev tools) to see why

File: src/sass/_variables.scss [hunk 2]
```diff
@@ -314,23 +314,24 @@ $aim-legs-xlrg-height-config: (
 
 // might have to update onHover in AgendaItemLegsFormReadOnly.jsx
 $aim-legs-sm-height-config-read-only: (
```
Comment (elizabeth-jimenez) [inline src/sass/_variables.scss:316]: try to understand how these are being used

**Comment by elizabeth-jimenez** [comment]
image
verified this is how it looks on dev branch

The only one that is messed up in dev is the Location/ Org, which is my bad, so you can go ahead and ignore that one; i'll open a PR for it shortly

**Comment by elizabeth-jimenez** [comment]
looking at patrick's commit, you may need to also update the hover in src/Components/Agenda/AgendaLeg/AgendaLeg.jsx

**Comment by elizabeth-jimenez** [comment]
since Action was moved to the top row, all of the variables in config need to be +1 - that should be all that's needed

**Comment by elizabeth-jimenez** [comment]
Checking
- [ ]  Agenda Item Maintenance Read (changed)
- [ ]  Agenda Item Maintenance Edit (changed)
- [ ]  Agenda Item history
- [ ]  Panel Meeting Agenda items
- [ ]  Print View of Panel (added)

**Pull request merged**